### PR TITLE
Replaces Half Life AI vox with an original set, adds many new lines.

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request

Removes all HL1 Vox lines, replaces them with a custom set I made. Given some of the questions I've received on discord I'm going to post the answers to some common questions I've received. 
**Q**. Why not just port TG's Vox?
**A**. They have a large amount of lines that cannot be used here, like "mimes" and "head of security" amongst others. The person who originally made the lines is long gone, so there's no recreating lines to replace the ones that I remove.
**Q**. How many lines did you add?
**A**. About 300.
**Q**. That's a lot, won't it take up too much space?
**A**. Currently _all_ the vox sounds collectively take 6mb, which is also how much TG's vox system takes up.
**Q**. This vox sounds nothing like the HL1 announcer, can't you just give him more lines?
**A**. Nobody knows how the Hl1 announcer was made, the secret sauce for making more lines is locked inside Gabe Newell's head. Thus I had to make a new announcer if I wanted to add more lines.
**Q**. Can you please add *insert line here*
**A**. Probably, make sure to comment it in the PR itself or post the line on discord where I can see it. Provided it's not too obscene or RP breaking I'll probably add it.
**Q**. Can I still use the vox to say "ass blast usa"?
**A**. Yes.

## Why It's Good For The Game

The current vox audio is very old and I have been getting a lot of feedback that there simply aren't enough words to describe things. On top of that it lacks many useful TGMC specific words like "Queen" or "resin" or most importantly "rouny".  ~~Also makes Gabe Newell unable to sue us~~ 

## Changelog
:cl:
soundadd: Added new original AI vox to replace the ancient half life vox.
soundadd: Added 300 extra lines for the AI vox system.
sounddel: Removed all Half Life vox
/:cl:

### CODE REVIEW

**Comment by Araulius** [comment]
But... but.. my ass blast usa! D:

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -553,19 +861,24 @@ GLOBAL_LIST_INIT(vox_sounds, list("!bloop" = 'sound/vox/!bloop.ogg',
 		"threat" = 'sound/vox/threat.ogg',
 		"three" = 'sound/vox/three.ogg',
 		"through" = 'sound/vox/through.ogg',
+		"throwing" = 'sound/vox/throwing.ogg',
 		"time" = 'sound/vox/time.ogg',
-		"to" = 'sound/vox/to.ogg',
+		"to" = 'sound/vox/two.ogg',
```
Comment (TiviPlus) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm:866]: ??

**Comment by TiviPlus** [comment]
How did you make these btw

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (RipGrayson) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm:866]: This was a size saving measure, I can generate different sound files if you need them to have seperate .ogg files.

**Comment by RipGrayson** [comment]
> How did you make these btw

That's a bit complicated, essentially it's the same (1996) TTS program that the original Hl1 announcer used. I couldn't replicate the original announcer so I settled for a new voice that honestly sounds more like HAL than the original. 
The audio effects are complicated, but involves vocoding the output of the TTS to an another ancient program and modifying the pitch/tone/reverb and applying a specific low pass filter.

**Review by TiviPlus — APPROVED** [review]

**Comment by RipGrayson** [comment]
I'm not doing anything particularly important here, feel free to merge this whenever.

**Pull request merged**